addons/website_sale/models/sale_order.py

```diff
@@ -133,7 +133,8 @@ def _website_product_id_change(self, order_id, product_id, qty=0):
                     discount = 0
                     pu = price
         else:
-            pu = product.price
+            # The price is already available without the need for calculations, so we take it directly from website_public_lst_price_usd
+            pu = product.website_public_lst_price_usd 
```

Comment:
@JulioSerna 

Could you review it, please?

I remember that you have 2 values to get price MXN and USD but it is determined by context

Is not?